Framework/Parallel/inc/MantidParallel/IO/EventParser.h

```diff
@@ -104,6 +78,12 @@ template <class TimeOffsetType> class EventParser {
   void doParsing(int32_t *event_id_start,
                  const TimeOffsetType *event_time_offset_start,
                  const Chunker::LoadRange &range);
+
+  void redistributeDataMPI();
```

Comment:
Were these moved out of testing scope because we only need to test against the resulting `EventList`?

Reply:
There was no test for the methods so making them private was ok. If we would later write tests for them, they can be made free functions again.